perf/perf_duplicate_probe.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2019 IBM
+# Author: Shirisha G <shiganta@in.ibm.com>
+
+import os
+import platform
+from avocado import Test
+from avocado import main
+from avocado.utils import distro, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class PerfProbe(Test):
+
+    def setUp(self):
+        '''
+        Install the basic packages to support perf and systemtap-sdt-devel
+        '''
+        # Check for basic utilities
+        smm = SoftwareManager()
+        detected_distro = distro.detect()
+        self.distro_name = detected_distro.name
```

Comment:
please use this as local scope variable as it not used any where aprt of setup sub routine

Reply:
and try to eliminate more variable defining as it increase grabage collector job

Reply:
sure Praveen